Framework/DataHandling/src/SaveReflectometryAscii.cpp

```diff
@@ -187,13 +186,26 @@ std::string SaveReflectometryAscii::sampleInfo(const std::string &logName) {
   return "Not defined";
 }
 
+/// Retrieve sample log unit
+std::string SaveReflectometryAscii::sampleUnit(const std::string &logName) {
+  auto run = m_ws->run();
+  try {
+    return " " +
+           boost::lexical_cast<std::string>(run.getLogData(logName)->units());
+  } catch (Exception::NotFoundError &) {
+  }
+  return "";
```

Comment:
Very minor point: I prefer you put the return for failure inside the catch block